### DISCUSSION TITLE
Fan Speed Issue with .187.0

### DISCUSSION THREAD

**Discussion by rhinehramirez** [seerge/g-helper#3060] [General]
Manually updated to .187.0 and now my fan speeds show 0 RPMs. After rebooting, the fan speeds show correctly, but after short amount of time (5-10 minutes), the issue persists. See screenshot. 
![Screenshot 2024-09-03 111451](https://github.com/user-attachments/assets/b7b82554-1f4c-4f19-ba9a-baa589460c46)

**Comment by seerge** [comment]
Closed as [multipost](https://github.com/seerge/g-helper/issues/3061)
